query-service/build.gradle.kts

```diff
@@ -23,6 +30,26 @@ dependencies {
   implementation("com.typesafe:config:1.4.0")
 
   runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.13.3")
+
+  implementation("com.typesafe:config:1.4.0")
+
+  integrationTestImplementation("org.junit.jupiter:junit-jupiter-api:5.6.2")
+  integrationTestImplementation("org.junit.jupiter:junit-jupiter-params:5.6.2")
+  integrationTestImplementation("org.junit.jupiter:junit-jupiter-engine:5.6.2")
+  integrationTestImplementation("org.testcontainers:testcontainers:1.15.2")
```

Comment:
If we are planning to start using test containers for all our integration tests eventually, we can add these dependencies in https://github.com/hypertrace/hypertrace-gradle-integration-test-plugin

Reply:
sure, I can do it after this pr is merged and test have been enabled?